docs/getting_started.md

```diff
@@ -132,36 +132,53 @@ Experiment with the graph range parameters and other settings.
 Let us make this more interesting and start some example targets for Prometheus
 to scrape.
 
-The Go client library includes an example which exports fictional RPC latencies
-for three services with different latency distributions.
+The next steps will depend what operating system you're running. You only need to
+follow one of them, and after that you can move on to the next section. By the end
+of the next sub-sections you should have example targets listening on
+http://localhost:8080/metrics, http://localhost:8081/metrics, and
+http://localhost:8082/metrics.
 
-Ensure you have the [Go compiler installed](https://golang.org/doc/install) and
-have a [working Go build environment](https://golang.org/doc/code.html) (with
-correct `GOPATH`) set up.
+### Setup node_exporter on Linux/Mac/Unix
 
-Download the Go client library for Prometheus and run three of these example
-processes:
+Start by downloading the latest version of node_exporter from the
+[Github Releases](https://github.com/prometheus/node_exporter/releases)
+* Look under assets and find the tar.gz file that most closely matches your operatings system.
+** If you don't know it's likely `amd64` for 64-bit and either `linux` or `darwin` (macOS).
 
 ```bash
-# Fetch the client library code and compile example.
-git clone https://github.com/prometheus/client_golang.git
-cd client_golang/examples/random
-go get -d
-go build
+# Go to the location of the download.
+cd ~/Downloads
+
+# <version> and <target> will depend on what you downloaded.
+tar -xzvf node_exporter-<version>.<target>.tar.gz
+cd node_exporter-<version>.<target>
 
 # Start 3 example targets in separate terminals:
-./random -listen-address=:8080
-./random -listen-address=:8081
-./random -listen-address=:8082
+./node_exporter --web.listen-address "127.0.0.1:8080"
+./node_exporter --web.listen-address "127.0.0.1:8081"
+./node_exporter --web.listen-address "127.0.0.1:8082"
 ```
 
-You should now have example targets listening on http://localhost:8080/metrics,
-http://localhost:8081/metrics, and http://localhost:8082/metrics.
+### Setup WMI exporter on Windows
```

Comment:
```suggestion
### Set up WMI exporter on Windows
```

Reply:
Ended up needing to remove section header :(